front/src/components/Tile.vue

```diff
@@ -53,7 +53,15 @@ import { TileValueUnit } from '@/store'
 
   import MonitororSubTile from '@/components/SubTile.vue'
   import MonitororTileIcon from '@/components/TileIcon.vue'
-  import {TileAuthor, TileConfig, TileState, TileStatus, TileType, TileValueUnit} from '@/store'
+  import {
+      DISPLAYABLE_SUBTILE_STATUS,
+      TileAuthor,
+      TileConfig,
+      TileState,
+      TileStatus,
+      TileType,
+      TileValueUnit,
```

Comment:
Why all those imports here?

Reply:
They were already there, they are just posted in multiline now